pkg/provider/kubernetes/auth_test.go

```diff
@@ -310,7 +310,7 @@ func TestSetAuth(t *testing.T) {
 				store: &esv1beta1.KubernetesProvider{
 					Server: esv1beta1.KubernetesServer{
 						URL:      "https://my.test.tld",
-						CABundle: []byte("1234"),
+						CABundle: []byte("bXljZXJ0ZGF0YQ=="),
```

Comment:
It might not be. I will play around with the base64 decoding part. Maybe I'm missing a call to that somewhere.

Reply:
Damn.

Auto decode doesn't work actually.

Because this:
```go
	out, err := base64.StdEncoding.DecodeString("1234")
	if err != nil {
		fmt.Println("err: ", err)
	}
```

does not error. It returns `�m�`.